gui/common_qt_functions.py

```diff
@@ -1,11 +1,16 @@
+# System imports
+import yaml
+import os
+from sys import platform
+
+# Import QT libraries
 from PySide6 import QtWidgets
 from PySide6.QtWidgets import QWidget
 from PySide6.QtCore import Qt, QSettings
-import yaml
-import os
 
 file_cache_eviction_choices = ['lru','lfu']
 libfusePermissions = [0o777,0o666,0o644,0o444]
+defaultFuseDir = '\Cloudfuse'
```

Comment:
These variables are only used inside widgetCustomFunctions. Should they be member fields of that class?

Reply:
I put it in the function itself